flash.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+"""script for easy update esp8266 firmware with esptool"""
```

Comment:
From PEP 257 -- Docstring Conventions:
The docstring of a script (a stand-alone program) should be usable as its "usage" message, printed when the script is invoked with incorrect or missing arguments (or perhaps with a "-h" option, for "help"). Such a docstring should document the script's function and command line syntax, environment variables, and files. Usage messages can be fairly elaborate (several screens full) and should be sufficient for a new user to use the command properly, as well as a complete quick reference to all options and arguments for the sophisticated user.

Please, add informatin about usage. Also add information that the script works with python version higher than 3.7.